msi/GitProduct.wxs

```diff
@@ -264,6 +264,8 @@
   </Fragment>
 
   <Fragment>
+    <SetProperty Id="ExecConfigureSslCAInfo" Value='"[#BashExe]" -c "git config -f \"/[#GfwGitConfig]\" http.sslCAinfo \"/[#ca_bundle.crt]\""' Sequence="execute" Before="ExecConfigureSslCAInfo" />
```

Comment:
Yes, ProgramData\Git\config should *not* be removed upon uninstallation.

As to handling `http.sslCAInfo` specially when it is found in a pre-existing ProgramData\Git\config, I am a bit hesitant to go that way: it sounds awfully complicated...

I'd rather just simply leave pre-existing settings alone and install that setting specifically into Git for Windows' own "system" config.